tools/xcms_macro/lib.r

```diff
@@ -149,7 +149,11 @@ getPlotChromPeakDensity <- function(xdata, param = NULL, mzdigit=4) {
 
     par(mfrow = c(3, 1), mar = c(4, 4, 1, 0.5))
 
-    group_colors <- brewer.pal(length(unique(xdata$sample_group)), "Set1")
+    if(length(unique(xdata$sample_group))<10){
+        group_colors <- brewer.pal(length(unique(xdata$sample_group)), "Set1")
+    }else{
+        group_colors <- hcl.colors(length(unique(xdata$sample_group)), palette="Dark 3")
```

Comment:
Is there a package we have to installed and load for hcl?

Reply:
If `r-rcolorbrewer` isn't used anymore (to be check), we can remote it as dependency, isn't it?

Reply:
`hcl.colors()` is a function of the `grDevices` package which is part of the "Standard R-base library" and thus is automatically charged in R, so no need to install or load any package. Note that this function requires R version >= 3.6.0 (currently used version in this XCMS Galaxy wrapper is 3.6.2). 

The `r-rcolorbrewer` package is still used: when class number is 9 or less, `brewer.pal()` is still used. This choice is based on: 
- the will to keep exact matching of colours with previous versions for classes up to 9
- the fact that for 9 or less classes, brewer.pal() generates more relevant colour palettes for qualitative colouring

Reply:
... It was easy to see, sorry for that!